packages/account/src/Sections/Profile/PhoneVerification/__test__/resend-code-timer.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import ResendCodeTimer from '../resend-code-timer';
+
+describe('ConfirmPhoneNumber', () => {
+    it('should disable button after its clicked', () => {
+        render(<ResendCodeTimer resend_code_text={'Resend code'} />);
+        const resend_button = screen.getByRole('button', { name: 'Resend code' });
```

Comment:
let's move both texts in tests to the variable and reuse them :)

Reply:
Will it improve readability?

Reply:
well, not exactly.
but if you extend test cases and reuse text again, definitely it will be more proper to reuse text.
if anything is repeated it is better create it once and reuse I believe.
you can ignore this comment :)

Reply:
Yes, if we add more testcases then sure 👌